### ISSUE TITLE
WP Media Library broken ACF Version 5.8.7

### ISSUE THREAD

**Issue by nothinghalosix** [funkjedi/acf-qtranslate#138]
Images in WP Library not display

https://support.advancedcustomfields.com/forums/topic/media-library-broken-after-update-to-acf-5-8-3-with-acf-qtranslate/

**Comment by herrvigg** [comment]
This is supposed to be solved since August 2019 in qTranslate-XT:
https://github.com/qtranslate/qtranslate-xt/issues/718

The current plugin has not been maintained and qTranslate-X is dead long ago. We have merged the last version in qTranslate-XT. I proposed several times to @funkjedi to collaborate but got no anwser from his side. We are evolving qTranslate slowly but surely and it's a lot of work (on my free time).

**Comment by fabienlege** [comment]
Ok ! @herrvigg , you here too ! ^^
i've just commented this issue on qtranslate-xt https://github.com/qtranslate/qtranslate-xt/issues/783 but error seam to come from this extension, not directly from qtranslate-xt. Is acf-qtranslate now integrated in qtranslate-xt and i can disable acf-qtranslate to my production website or it's two different things and acf-qtranslate must remain activated ?

thanks

**EDIT :** this fork seam to do the job : https://github.com/afonsoduarte/acf-qtranslate

**Comment by herrvigg** [comment]
Yes that's it. But i really recommend you to switch to [qTranslate-XT](https://github.com/qtranslate/qtranslate-xt) despite the fact it's not an official plugin. The old qTranslate-X is completely broken, i don't even understand how people still manage to use it.

This plugin here for ACF is a great one and i can appreciate all the nice work being done here. But since we are preparing a new migration and refactor a bunch of core functions i decided to handle the add-ons directly in the main project. Having external plugins is not practical when making such evolutions, and in the case of qTranslate which has been a zombie repo for such a long time, it's really not viable anymore. A lot of breaking changes are going to happen so you will understand better. Now the problem would be to maintain the two things but this is why i proposed a collaboration. In any case the new repo will be open-source as well so any contributor is much welcome already now in qT-XT.
